crates/jarl/tests/integration/no_default_exclude.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::Command;
+use tempfile::TempDir;
+
+use crate::helpers::CommandExt;
+use crate::helpers::binary_path;
+
+#[test]
+fn test_no_default_exclude() -> anyhow::Result<()> {
+    let directory = TempDir::new()?;
+    let directory = directory.path();
+
+    let test_path = "cpp11.R";
+    let test_contents = "
+x = 1
+y <- 2
+3 -> z
+";
```

Comment:
```suggestion
    let test_contents = "x = 1";
```

We just need one violation here.